### ISSUE TITLE
Long drives issue

### ISSUE THREAD

**Issue by teliot** [jasonrohrer/OneLifeData7#1220]
So here I am taking a leisurely scenic drive through the woods for HOURS east to go see a buddies monument. In the middle of this drive my sports car stopped going vroom. The support team pulled into action flying me new kero and fresh craving foods. The issue is not so much that sitting in a car for a straight hour driving runs out of fuel, but how hard the client struggles to get out of said car once you realize the game is no longer making brpbrpbrpbrpbrpbrpbrpbrpbrp sound.

**Comment by jasonrohrer** [comment]
Delivery truck also works same way, also with no way to set it down after it runs out!

Crude car looks like it gives you 4 seconds to set it down when it runs out (in 10 minutes)

**Comment by jasonrohrer** [comment]
Actually, in testing this, I'm not seeing the problem.

Yes, the sports car, and delivery truck, runs out of fuel after an hour of driving...

But then you just right-click bare ground nearby, and set it down.

Now... I guess there could be a rare situation where you stop driving surrounded by 4 stones or something?  I will test that next.

**Comment by teliot** [comment]
Thanks! I didn't understand I was out of fuel and trying to drive really wigged out the game client. I ended up restarting the game client twice and trying to drive, but once I restarted and got out it all made sense when I couldn't get back in. This solves this, and even if it dumps me out in an unusable biome I would be in the same situation as if I had to get out at the same spot from running out of fuel. I am thinking this change is best.
